lib/Compilation.js

```diff
@@ -1067,13 +1072,14 @@ class Compilation {
 			},
 			err => {
 				if (this.bail) {
```

Comment:
@sokra is `if (this.bail)` correct here? why not `if (err && this.bail)`?

Reply:
Yep. Same concern for me.

Reply:
yep, seem to be wrong...